src/android/com/outsystems/firebase/cloudmessaging/OSFirebaseCloudMessaging.kt

```diff
@@ -29,16 +32,19 @@ class OSFirebaseCloudMessaging : CordovaImplementation() {
 
     private var deviceReady: Boolean = false
     private val eventQueue: MutableList<String> = mutableListOf()
-    private var notificationPermission = OSNotificationPermissions()
+
+    private val gson: Gson = GsonBuilder().excludeFieldsWithoutExposeAnnotation().create()
+
+    private var flow: MutableSharedFlow<OSFCMPermissionEvents>? = null
 
     companion object {
         private const val CHANNEL_NAME_KEY = "notification_channel_name"
         private const val CHANNEL_DESCRIPTION_KEY = "notification_channel_description"
         private const val ERROR_FORMAT_PREFIX = "OS-PLUG-FCMS-"
         private const val NOTIFICATION_PERMISSION_REQUEST_CODE = 123123
-        private const val NOTIFICATION_PERMISSION_SEND_LOCAL_REQUEST_CODE = 987987
         const val FCM_EXPLICIT_NOTIFICATION = "com.outsystems.fcm.notification"
         const val GOOGLE_MESSAGE_ID = "google.message_id"
+        const val POST_NOTIFICATIONS_PERMISSION = "android.permission.POST_NOTIFICATIONS"
```

Comment:
`Manifest.permission.POST_NOTIFICATIONS` can be used so this `const val` can be avoided.